src/coreComponents/finiteVolume/SurfaceElementStencil.hpp

```diff
@@ -475,6 +475,7 @@ SurfaceElementStencilWrapper::
       real64 const harmonicWeight   = t0*t1 / sumOfTrans;
       real64 const arithmeticWeight = 0.25 * (t0+t1);
 
+      m_meanPermCoefficient = 0;
```

Comment:
@Guotong-Ren not sure i fully understand it here - you want to switch from harmonic to arithmetic? does it help with convergence?

Reply:
> this allows the fluid going into the newly added fracture elements. Otherwise, harmonic average will always result 0 trans if new fracture element's aperture = 0

Very good point, @CusiniM can we frame this change properly?

> BTW, i have a local version merged with Matteo's latest changes on total stress sif. I will push it to this branch.

Thanks and let me know if you will get any conflicts. Sorry for messing up this branch...

Reply:
In reality, the aperture can never be 0. The hydraulic aperture is the one used here and it's never 0.

Reply:
I think what you assume is the default aperture is a number larger than zero?

Reply:
Somehow that change seems to fix failing case from docs: https://github.com/GEOS-DEV/GEOS/issues/2632
@CusiniM and @Guotong-Ren could you please double check?